contrib/platform/src/com/sun/jna/platform/win32/WinPerf.java

```diff
@@ -0,0 +1,451 @@
+/* Copyright (c) 2015 Goldstein Lyor, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.sun.jna.Structure;
+import com.sun.jna.platform.win32.WinBase.SYSTEMTIME;
+import com.sun.jna.platform.win32.WinNT.LARGE_INTEGER;
+
+/**
+ * Various performance counters structures and definitions
+ * @author Lyor Goldstein
+ * @see <A HREF="https://msdn.microsoft.com/en-us/library/windows/desktop/aa373093(v=vs.85).aspx">Performance Counters Structures</A>
+ */
+public interface WinPerf {
+    /**
+     * Describes the performance data block that you queried
+     * @see <A HREF="https://msdn.microsoft.com/en-us/library/windows/desktop/aa373157(v=vs.85).aspx">PERF_DATA_BLOCK</A>
+     */
+    public class PERF_DATA_BLOCK extends Structure {
+        public char[]        Signature=new char[4];
+        public int           LittleEndian;
+        public int           Version;
+        public int           Revision;
+        public int           TotalByteLength;
+        public int           HeaderLength;
+        public int           NumObjectTypes;
+        public int           DefaultObject;
+        public SYSTEMTIME    SystemTime=new SYSTEMTIME();
```

Comment:
You might want to auto-format this file too, there should be a space around the `=`.